src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesBackupProtectionPolicy.md

```diff
@@ -135,7 +135,7 @@ Currently, only AzureVM, AzureFiles is supported.
 Type: System.Nullable`1[Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.Models.WorkloadType]
 Parameter Sets: WorkloadParamSet, WorkloadBackupManagementTypeParamSet
 Aliases:
-Accepted values: AzureVM, AzureSQLDatabase, AzureFiles, MSSQL
+Accepted values: AzureVM, AzureSQLDatabase, AzureFiles, MSSQL, FileFolder
```

Comment:
FileFolder is not supported. please remove this

Reply:
removed